tests/e2e/pageobjects/ide/Ide.ts

```diff
@@ -219,31 +270,30 @@ export class Ide {
     }
 
     async closeAllNotifications() {
-        const notificationLocator: By = By.css('.theia-Notification');
-
+        const statusBarLocator: By = By.id('theia-statusBar');
```

Comment:
The statusBarLocator seems useless for me. It can be part of statusBarNotificationsLocator IMO.

Reply:
rewritten it completely, it's deleted now.